scripts/cibuild/CreateEndToEndTestPackage.ps1

```diff
@@ -61,6 +61,10 @@ try {
     Write-Verbose "Copying test extension from '$TestExtension' to '$WorkingDirectory'"
     Copy-Item $TestExtension $WorkingDirectory
 
+    $GeneratePackagesUtil = Join-Path $NuGetRoot "artifacts\GenerateTestPackages\${ToolsetVersion}.0\bin\${Configuration}\net46"
```

Comment:
It's the same output path structure as in `$TestExtension` above. `$ToolsetVersion` here has a different meaning from `build.ps1`.